### DISCUSSION TITLE
Seprate Client\Server machines

### DISCUSSION THREAD

**Discussion by malakibrahim1** [appneta/tcpreplay#857] [Q&A]
I wonder if there is a way to install TCPREPLY on a VM that acts as a client and a separate VM as a responding server. I have a SASE solution where security takes place in the cloud, and I have two remote sites where each site will host a client or server VM to test the SASE solution.

**Comment by fklassen** [comment]
Tcpreplay is a packet producer. I doesn't read packets. So that scenerio is not possible.
